docs/reference/api/drivers/drivers.md

```diff
@@ -1,3 +1,25 @@
 ## Drivers overview
 
-Driver APIs include analog and digital inputs and outputs on development boards, as well as digital interfaces, which allow your board to interface with a computer or external devices. With these drivers, you can read or set the voltage of analog pins, control digital pins individually or as a grouped value and trigger an event when a digital input pin changes value. You can also control the frequency and mark to space ratio of a digital pulse wave (or train).
+Drivers provide access to general purpose microcontroller hardware. Drivers typically don't require software stacks but can be the signaling foundation of protocol stacks and networking stacks. 
```

Comment:
Drivers typically access a microcontroller's hardware peripherals directly, and may form the lowest lever of a protocol or network stack.